File: 프로그래머스/2/70129. 이진 변환 반복하기/이진 변환 반복하기.py

def solution(s):
    zeroCount = 0;
    count = 0;
    answer = [];
    while (s != '1') : 
        zeroCount += s.count('0');
        s = s.replace('0','');
        s = bin(len(s))[2:]     
        count +=1;
        
    answer = [count, zeroCount]    

    
    return answer   

File: 프로그래머스/2/70129. 이진 변환 반복하기/이진 변환 반복하기.js
function solution(s) {
    var answer = [];
    let a = 0
    let b = 0
    while (s != '1') {
        a += s.length - s.replaceAll('0','').length
        s = s.replaceAll('0','') 
        s = s.length.toString(2)
        b+=1
    }
    answer.push(b)
    answer.push(a)
    return answer;
}